README.md

```diff
@@ -216,9 +219,9 @@ I would like to thank...
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
+- **[How do you collect data after the contact form is filled?]**
 
-  - [Answer_1]
+  - [I used formspree. You can create an account  with formspree this will help you track if your contact form is valid. And the goo news is that formspree is totally free.]
 
 - **[Question_2]**
 
```

Comment:
- [ ] When I click the link to the MIT on this file, I get directed to this error page. 

![image](https://user-images.githubusercontent.com/77106962/207182052-498886fb-a039-4fdb-b9ab-b3a2ef127b97.png)

You can learn how to use one of the free open source Licenses on Github by clicking this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)